### PR TITLE
style fixes: badges & search close icon

### PR DESCRIPTION
**Pull request by kdembler**

### CODE REVIEW

File: packages/atlas/src/components/_buttons/Button/Button.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         textOnly={textOnly}
         variant={variant}
         iconOnly={iconOnly}
+        data-badge={badge}
```
Comment (eldiegod) [inline packages/atlas/src/components/_buttons/Button/Button.tsx:33]: we should add `data-badge` as an emotion prop in this component

**Review by eldiegod — APPROVED** [review]
Looks good added a suggestion

File: packages/atlas/src/components/_buttons/Button/Button.tsx (hunk 1 above)
Comment (eldiegod) [reply packages/atlas/src/components/_buttons/Button/Button.tsx:33]: possibly even make it a string and use `formatNumberShort` so it doesn't break with big numbers

Comment (kdembler) [reply packages/atlas/src/components/_buttons/Button/Button.tsx:33]: Added to props, made it to always be string, let's leave it up to consumer to format this string if needed

**Review by eldiegod — APPROVED** [review]

**Comment by eldiegod** [comment]
There is a type error on filters bar badge due to now being a string but besides LGTM

File: packages/atlas/src/components/_buttons/Button/Button.tsx (hunk 1 above)
Comment (kdembler) [reply packages/atlas/src/components/_buttons/Button/Button.tsx:33]: Okay, changed that back to allow number as well, it's up to the consumer, doesn't make a difference for the component

**Pull request merged**